extensions/default/src/DicomWebDataSource/utils/clientManager.ts

```diff
@@ -0,0 +1,249 @@
+import { api } from 'dicomweb-client';
+import StaticWadoClient from './StaticWadoClient';
+import dcm4cheeReject from '../dcm4cheeReject';
+import { errorHandler, utils } from '@ohif/core';
+
+/**
+ * This object plays the central role in OHIF's multiple server handling ability.
+ * It stores all servers configurations and handles all request headers generations
+ */
+interface ConfigToAdd {
+  url: string;
+  qidoRoot: string;
+  wadoRoot: string;
+  staticWado: boolean;
+  singlepart: boolean;
+  name: string;
+  qidoConfig: {
+    url: string;
+    staticWado: boolean;
+    singlepart: boolean;
+    headers: object;
+  };
+  wadoConfig: {
+    url: string;
+    staticWado: boolean;
+    singlepart: boolean;
+    headers: object;
+  };
+  qidoDicomWebClient: api.DICOMwebClient | StaticWadoClient;
+  wadoDicomWebClient: api.DICOMwebClient | StaticWadoClient;
+}
```

Comment:
I expected this interface has all the keys in our default.js

```js
configuration: {
friendlyName: 'AWS S3 Static wado server',
name: 'aws',
wadoUriRoot: 'https://d33do7qe4w26qo.cloudfront.net/dicomweb',
qidoRoot: 'https://d33do7qe4w26qo.cloudfront.net/dicomweb',
wadoRoot: 'https://d33do7qe4w26qo.cloudfront.net/dicomweb',
qidoSupportsIncludeField: false,
imageRendering: 'wadors',
thumbnailRendering: 'wadors',
enableStudyLazyLoad: true,
supportsFuzzyMatching: false,
supportsWildcard: true,
staticWado: true,
singlepart: 'bulkdata,video',
// whether the data source should use retrieveBulkData to grab metadata,
// and in case of relative path, what would it be relative to, options
// are in the series level or study level (some servers like series some study)
bulkDataURI: {
  enabled: true,
  relativeResolution: 'studies',
},
omitQuotationForMultipartRequest: true,
},
```

Reply:
I put the most important ones only. Should I put all?

Reply:
yes please